x-pack/tasks/download_chromium.ts

```diff
@@ -21,5 +21,6 @@ export const downloadChromium = async () => {
     log: consoleLogger('log'),
   };
 
-  await download(logger);
+  // Download Chromium for all platforms
+  await Promise.all(paths.packages.map((pkg) => download(pkg, logger)));
```

Comment:
In dev mode and at build time, the `download` function was unintentionally downloading a Chromium package for every platform and architecture. That change impacted this code, which intentionally downloads every package. This code is called from CI and is used to avoid downloading the browsers for every CI job

Reply:
Trying to understand this. We want to download all permutations of Chromium on CI, but not in dev mode or at build? Where is this script called from?

Reply:
@vadimkibana it is called from this script: https://github.com/elastic/kibana/blob/main/.ci/packer_cache_for_branch.sh#L31